src/deployment/android.md

```diff
@@ -95,64 +95,66 @@ To find out the latest version, visit [Google Maven][].
 +<style name="NormalTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-## Signing the app
+## Sign the app
 
-To publish on the Play Store, you need to give your app a digital
-signature. Use the following instructions to sign your app.
+To publish on the Play Store, you sign your app with a digital signature.
 
-On Android, there are two signing keys: deployment and upload. The end-users 
-download the .apk signed with the 'deployment key'. An 'upload key' is used to 
-authenticate the .aab / .apk uploaded by developers onto the Play Store and is 
-re-signed with the deployment key once in the Play Store.
-* It's highly recommended to use the automatic cloud managed signing for
-  the deployment key. For more information,
-  check out the [official Play Store documentation][].
+Android uses two signing keys: _upload_ and _app signing_.
+
+* Developers upload an `.aab` or `.apk` file signed with an _upload key_ to the Play Store.
+* The end-users download the `.apk` file signed with an _app signing key_.
+
+To create your app signing key, use Play App Signing
+as described in the [official Play Store documentation][].
```

Comment:
I'm a bit confused here as this text says to use Play App Signing to create my key, then the next category again gives me two options to create my key including that one. Perhaps this note can be dropped?

Reply:
You need to create the upload key and keystore first, then you can use the Play App Signing. This does explain that.